src/Ocelot/RateLimiting/DistributedCacheRateLimitStorage.cs

```diff
@@ -1,13 +1,19 @@
 ﻿using Microsoft.Extensions.Caching.Distributed;
 using Newtonsoft.Json;
 
-namespace Ocelot.RateLimit
+namespace Ocelot.RateLimiting
 {
-    public class DistributedCacheRateLimitCounterHandler : IRateLimitCounterHandler
+    /// <summary>
+    /// Custom storage based on a distributed cache of a remote/local services.
+    /// </summary>
+    /// <remarks>
+    /// See the <see cref="IDistributedCache"/> interface docs for more details.
+    /// </remarks>
+    public class DistributedCacheRateLimitStorage : IRateLimitStorage
     {
         private readonly IDistributedCache _memoryCache;
```

Comment:
Not part of the PR, but since you are refactoring it, the code could be simplified.
```c#
public void Set(string id, RateLimitCounter counter, TimeSpan expirationTime) => 
    _memoryCache.SetString(id, JsonConvert.SerializeObject(counter), new DistributedCacheEntryOptions().SetAbsoluteExpiration(expirationTime));

public bool Exists(string id) => !string.IsNullOrEmpty(_memoryCache.GetString(id));

public void Remove(string id) => _memoryCache.Remove(id);
```

Reply:
Fixed in commit https://github.com/ThreeMammals/Ocelot/pull/1592/commits/2b0639cc3adfeffc812f785d39e9e72fb052057f ✔️